test/GCHP/intTestCreate.sh

```diff
@@ -31,173 +27,148 @@
     exit 1
 fi
 
 # Environment file
-envFile=${2}
+envFile="${2}"
 if [[ "x${envFile}" == "x" ]]; then
     echo "ERROR: The enviroment file (w/ module loads) has not been specified!"
     exit 1
 fi
-if [[ ! -f ${envFile} ]]; then
+if [[ ! -f "${envFile}" ]]; then
     echo "ERROR: The enviroment file is not a valid file!"
     exit 1
 fi
 
 # Run a short integration test?
-short=${3}
+quick="${3}"
 
 #=============================================================================
 # Global variable and function definitions
 #=============================================================================
 
 # Current directory
-testDir=$(pwd -P)
-cd ${testDir}
-
-# Top-level GEOS-Chem directory
-cd ../..
-geosChemDir=$(pwd -P)
+thisDir=$(pwd -P)
+cd ${thisDir}
 
 # GCClassic superproject directory
-cd ../../../..
+cd ../../../../../..
 superProjectDir=$(pwd -P)
-cd ${superProjectDir}
+cd "${superProjectDir}"
 
-# Directory where the run creation scripts are found
-runDir=${geosChemDir}/run/GCHP
+# GEOS-Chem and HEMCO submodule directories
+geosChemDir="${superProjectDir}/src/GCHP_GridComp/GEOSChem_GridComp/geos-chem"
+hemcoDir="${superProjectDir}/src/GCHP_GridComp/HEMCO_GridComp/HEMCO"
 
-# Load file with utility functions to setup configuration files
-. ${geosChemDir}/test/shared/commonFunctionsForTests.sh
+# Get the Git commit of the superproject and submodules
+head_gchp=$(export GIT_DISCOVERY_ACROSS_FILESYSTEM=1; \
+	    git -C "${superProjectDir}" log --oneline --no-decorate -1)
+head_gc=$(export GIT_DISCOVERY_ACROSS_FILESYSTEM=1; \
+	  git -C "${geosChemDir}" log --oneline --no-decorate -1)
+head_hco=$(export GIT_DISCOVERY_ACROSS_FILESYSTEM=1; \
+	   git -C "${hemcoDir}" log --oneline --no-decorate -1)
 
-# Get the absolute path of the root folder
-root=$(absolute_path ${root})
-
-# Log file
-log=${root}/logs/createIntTests.log
+# Source the script containing utility functions and variables
+. "${geosChemDir}/test/shared/commonFunctionsForTests.sh"
 
 # Echo header
 printf "${SEP_MAJOR}\n"
-printf "Creating GCHP Integration Tests\n"
+printf "Creating GCHP Integration Tests\n\n"
+printf "GCHP      #${head_gchp}\n"
+printf "GEOS_Chem #${head_gc}\n"
+printf "HEMCO     #${head_hco}\n"
 printf "${SEP_MAJOR}\n"
 
 #=============================================================================
 # Initial setup of integration test directory
 #=============================================================================
 
 # Create integration test root folder if it doesn't exist
-if [[ ! -d ${root} ]]; then
-    mkdir -p ${root}
-fi
+itRoot=$(absolute_path "${itRoot}")
+[[ ! -d "${itRoot}" ]] && mkdir -p "${itRoot}"
 
-# Get the absolute path of the root folder again;
-# now that the folder exists
-root=$(absolute_path ${root})
+# Get absolute path of the environment file
+envFile=$(absolute_path "${envFile}")
 
-# Remove run directories in the test folder
-cleanup_files ${root}
+# Remove run directories in the te folder
```

Comment:
Fix "te" typo. It used to be "test."

Reply:
Fixed in commit 9f68416bf630f7e303e376f1868a0bd7121bcf53